bigquery_etl/metadata/parse_metadata.py

```diff
@@ -146,7 +207,26 @@ def write(self, file):
                 if label_value == "":
                     metadata_dict["labels"][label_key] = True
 
-        file.write_text(yaml.dump(metadata_dict))
+        # Use literal yaml representation for descriptions
+        class Literal(str):
+            pass
+
+        def literal_presenter(dumper, data):
+            return dumper.represent_scalar("tag:yaml.org,2002:str", data, style="|")
+
+        yaml.add_representer(Literal, literal_presenter)
```

Comment:
Does this affect `yaml` library behavior globally? Should this be happening once in global scope rather than nested here?